pages/linux/phar.md

```diff
@@ -0,0 +1,35 @@
+# phar
+
+> Create, update or extract PHP archives (PHAR).
+
+- Add space-separated files or directories to a Phar file:
+
+`phar add -f {{path/to/phar}} {{files_or_directories}}`
+
+- Delete the specified file or directory from a Phar file:
```

Comment:
Should we move the `list -f` above this so that a user can find out what files are there before deleting something ?

Reply:
I guess that makes sense :+1: